api/config.py

```diff
@@ -211,53 +211,67 @@ class Configuration(CoreConfiguration):
         }
     ]
 
+    # The "level" property determines which admins will be able to modify the setting.  Level 1 settings can be modified by anyone.
+    # Level 2 settings can be modified only by library managers and system admins (i.e. not by librarians).  Level 3 settings can be changed only by system admins.
+    # If no level is specified, the setting will be treated as Level 1 by default.
+    ALL_ACCESS = 1
+    SYS_ADMIN_OR_MANAGER = 2
+    SYS_ADMIN_ONLY = 3
+
```

Comment:
You should be able to import these from core/config.py rather than defining them again -- that way there's no risk they'll get out of sync.